workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentConfigurations.cs

```diff
@@ -0,0 +1,163 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Improbable.Gdk.DeploymentManager
+{
+    public class DeploymentConfig
+    {
+        /// <summary>
+        ///     The name of the deployment to launch.
+        /// </summary>
+        public string Name;
+
+        /// <summary>
+        ///     The name of the SpatialOS project to launch in.
```

Comment:
```suggestion
        ///     The name of the SpatialOS project to launch the deployment in.
```

Reply:
I've removed this field while doing the UI changes.